src/mmw/js/src/analyze/views.js

```diff
@@ -466,6 +469,104 @@ var PointSourceTableView = Marionette.CompositeView.extend({
     }
 });
 
+var CatchmentWaterQualityTableRowView = Marionette.ItemView.extend({
```

Comment:
I feel like it makes sense to do the splitting of this file in tandem with the splitting of the UI in #1525